MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include README.md
+include LICENSE
```

Comment:
We should add the following:

```
graft tests
graft scripts
global-exclude __pycache__
global-exclude *.py[co]
```

This removes `tests*` and `scripts*` from the `sdist`.